hbase-server/src/test/java/org/apache/hadoop/hbase/regionserver/TestOpenRegionFailedMemoryLeak.java

```diff
@@ -60,6 +63,14 @@ public class TestOpenRegionFailedMemoryLeak {
 
   private static HBaseTestingUtil TEST_UTIL = new HBaseTestingUtil();
 
+  @BeforeClass
+  public static void startCluster() throws Exception {
+    Configuration conf = TEST_UTIL.getConfiguration();
+
+    // Enable sanity check for coprocessor
+    conf.setBoolean(TableDescriptorChecker.TABLE_SANITY_CHECKS, true);
```

Comment:
Why we need this change?

Reply:
Oh, it's due to the fact that this test checks region open [is failing due to invalid loaded coprocessor at line 90](https://github.com/apache/hbase/pull/4533/files#diff-38ced1e200aafb2db3ee4f242681b31ee6a379f31ca238bd42ac86a7eaf46463R90) below, before this change it was always true (since the coprocessor loading was independent of sanity check). However after this change, we have made it configurable based on the sanity check config ([which is always false in hbase-server's test related hbase-site.xml](https://github.com/apache/hbase/blob/master/hbase-server/src/test/resources/hbase-site.xml#L154)), so we need to explicitly enable sanity check in the test setup

Reply:
OK, seems to have some side effects. Need to fill a release note.